mobx_codegen/lib/src/type_names.dart

```diff
@@ -96,6 +96,10 @@ class LibraryScopedNameFinder {
             type is TypeParameterType ||
             // If the type lives in this library, no prefix necessary
             typeElement.library == library) {
+      // TODO(shyndman): This ignored deprecation can be removed when we
```

Comment:
What's preventing from increasing the min version ?

Reply:
Let's avoid another minor version bump. We should work on stabilizing a version for people.